### PR TITLE
Add timer to avoid hit pantahub every second with device meta

### PR DESCRIPTION
**Pull request by fortizc**
Add "metadata.devmeta.interval" to allow set the device meta timer

### CODE REVIEW

File: metadata.c [hunk 1]
```diff
@@ -641,6 +645,14 @@ int pv_metadata_upload_devmeta(struct pantavisor *pv)
 	struct dl_list *head = NULL;
 	int json_avail = 0, ret = 0;
 	struct buffer *buffer = NULL;
+
+	struct timer_state st = timer_current_state(&pv->metadata->devmeta_tm);
+	if (!st.fin)
+		return 0;
+
+	timer_start(&pv->metadata->devmeta_tm,
+							pv_config_get_metadata_devmeta_interval(), 0, RELATIV_TIMER);
```
Comment (asac) [inline metadata.c:654]: indent looks off...

File: metadata.c [hunk 2]
```diff
@@ -630,6 +631,9 @@ int pv_metadata_init_devmeta(struct pantavisor *pv)
 	}
 	pv_buffer_drop(buffer);
 	pv->metadata->devmeta_uploaded = false;
+	timer_start(&pv->metadata->devmeta_tm,
+							pv_config_get_metadata_devmeta_interval(), 0, RELATIV_TIMER);
```
Comment (asac) [inline metadata.c:635]: indent looks off ...

**Comment by anibalportero** [comment]
Looks good to me. When you add or modify something in the config, you also have to update this file in docs. So, please, create a MR with the new key in the table:

https://docs.pantahub.com/pantavisor-configuration/
https://gitlab.com/pantacor/docs

File: metadata.c (hunk 1 above)
Comment (fortizc) [reply metadata.c]: Fixed

File: metadata.c (hunk 2 above)
Comment (fortizc) [reply metadata.c]: Fixed

**Comment by fortizc** [comment]
> Looks good to me. When you add or modify something in the config, you also have to update this file in docs. So, please, create a MR with the new key in the table:
> 
> https://docs.pantahub.com/pantavisor-configuration/ https://gitlab.com/pantacor/docs

Ok, cool

**Pull request merged**